transfer/policy.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Id": "transfer-bucket-policy",
+  "Statement": [
+    {
+      "Action": "*",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Resource": "arn:aws:s3:::${bucket}/*",
+      "Sid": "transfer-bucket-policy"
+    }
+  ],
+  "Version": "2012-10-17"
+}
```

Comment:
`\n`

Reply:
this doesn't seem right, seems to give any AWS account access to any object